src/System.Device.Gpio.Tests/RaspberryPiDriverTests.Linux.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace System.Device.Gpio.Tests
 {
+    [Trait("requirement", "root")]
```

Comment:
Currently all tests run as root already

Reply:
In the CI yes but not when I test locally.